ibmcloud/terraform/cluster/scripts/keygen.sh

```diff
@@ -68,7 +70,12 @@ for host in "${hosts[@]}"; do
         trap stop 0
 
         ssh "${opts[@]}" -o ExitOnForwardFailure=yes -S "$ssh_ctl_sock" -L "$port:$host:22" -M -f -N -l root "$bastion"
-        ssh-copy-id "${opts[@]}" -i "$key.pub" -p "$port" root@localhost
+        if [[ "${os}" = "Darwin" ]]; then
+            ssh-copy-id -f "${opts[@]}" -i "$key.pub" -p "$port" root@localhost
+        else
+            ssh-copy-id "${opts[@]}" -i "$key.pub" -p "$port" root@localhost
+        fi
+        
```

Comment:
```suggestion
        [[ $(uname) = "Darwin" ]] && force_option="-f"
        echo "ssh-copy-id ${force_option:-} \"${opts[@]}\" -i \"$key.pub\" -p \"$port\" root@localhost"
```
There isn't anything wrong with your code, but to avoid the duplication and make it a bit more succinct, you could consider something like this?

Reply:
Thanks. I attempted something like that but the solution I tried broke ssh-copy-id when the force_option option was not set. The ` ${force_option:-}` part seems to be the magic that handles the case where `force_option` is undefined correctly.